extendr-api/src/robj_ndarray.rs

```diff
@@ -127,3 +158,28 @@ fn test_from_robj() {
         // assert_eq!(mx[[3, 1]], 8);
     }
 }
+
+#[test]
+fn test_to_robj() {
+    test! {
+    assert_eq!(
+        &Robj::from(array![1., 2., 3.]),
+        &R!("array(c(1, 2, 3))").unwrap()
+    );
+
+    assert_eq!(
+        &Robj::from(Array::from_shape_vec((2, 3), vec![1., 2., 3., 4., 5., 6.]).unwrap()),
+        &R!("matrix(c(1, 2, 3, 4, 5, 6), nrow=2, byrow=T)").unwrap()
+    );
+
+    assert_eq!(
+        &Robj::from(Array::from_shape_vec((3, 2).f(), vec![1., 2., 3., 4., 5., 6.]).unwrap()),
+        &R!("matrix(c(1, 2, 3, 4, 5, 6), nrow=2, byrow=T)").unwrap()
+    );
+
+    assert_eq!(
+        &Robj::from(array![[[1, 5], [3, 7]], [[2, 6], [4, 8]]]),
+        &R!("array(1:8, dim=c(2, 2, 2))").unwrap()
+    );
```

Comment:
It would be nice to have a test for something like `array(1:6, c(1, 2, 3))`, i.e. when the size of one of the dimensions is 1.

Reply:
Fair enough, will do.

Reply:
Fair enough, will do.